cypress/e2e/automatic-layout.feature

```diff
@@ -5,13 +5,54 @@ Feature: Test automatic layout for the graph
     And I visit the "/"
 
   Scenario: Auomatic layout should have a (deterministic) given result
+    # Initial positions
+    Then I expect "#__shadowRoot" to be at position 0,0
```

Comment:
Split into two scenario, one that check all is rerranged and another to check only component in container is rearranged.

Reply:
done